.gitignore

```diff
@@ -1,2 +1 @@
-~$*.xlsx
-/.project
+.project
```

Comment:
Die Änderung verstehe ich nicht so ganz, Excel-Temp-Dateien würde ich schon noch ignorieren wollen und .project nur im Root-Verzeichnis zu ignorieren finde ich auch sinnvoll.

Reply:
Ich dachte das zunächst auch, aber die jeweiligen Produktseiten verweisen auf unterschiedliche Datenblätter. Auch gibt es ein "complete" datasheet und ein "datasheet". Von mir aus kannst du du auch das vom 644er weglassen, das würde auch Speicherplatz sparen, aber schau besser nochmal in die pdfs rein. Über die Temp-Datei hatte ich mich auch gewundert ehrlich gesagt weil ich ähnliches in den anderen Repos nicht gesehen hatte. Die .project sind in den anderen Repos auch allgemein ignoriert. Daran hatte ich mich orientiert und das bei der Gelegenheit angepasst. Sollte man vllt. einheitlich machen, aber wieso sollten .project setting files überhaupt in Unterverzeichnissen auftauchen? Die werden doch _immer_ im Root-Verzeichnis erzeugt.

Reply:
Ah okay, ich hatte auch nicht reingeschaut, mich nur über die Dateigröße gewundert und dann gesehen, dass die jetzt ein kombiniertes Datenblatt haben. Dann schaue ich da noch mal rein.

Na ja in den anderen Repos gibt es ja auch keine Excel-Dateien ;-)
Ja eben weil sie Immer nur im Root auftauchen, braucht man sie auch nur dort zu ignorieren. Dann fällt es auch auf, falls sie (z.B. fälschlicherweise) doch mal woanders vorkommen. Oder wenn ein anderes Programm auch .project Dateien erzeugt, die man dann vielleicht auch einchecken will.

Reply:
Alles klar, dann würde ich das beim jeweils nächsten Commit bei den anderen Repos aber auch nachziehen.